assets/script.js

```diff
@@ -19,7 +19,29 @@ function startTimer(seconds) {
       timer--;
     }
   }, 1000);
+
+}
+
+function displayContent(type) {
```

Comment:
This function doesn't really need to know what type is being passed in, it doesn't matter if it's a joke or quote, it simply needs to display the content on the page, so pass in something like 'str' instead and update using the the reference you establish at the top of the screen.